### PR TITLE
add script for postgres database setup without docker

### PR DESCRIPTION
**Pull request by SimonSchneiderHSM**
If you want to setup the postgres database for the demo without using docker, you can use the script.

It assumes that the "processengine" database and the role "admin" does not yet exist.

Pre-requisite: postgres installed and running.

### CODE REVIEW

File: database/setup_database_demo/create_role_and_db.sql [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+SET default_tablespace = '';
+
+SET default_with_oids = false;
+
+CREATE ROLE admin WITH PASSWORD 'admin';
+ALTER ROLE admin WITH LOGIN;
+ALTER ROLE admin WITH CREATEDB;
+
+SET ROLE admin;
+CREATE DATABASE processengine;
```
Comment (heikomat) [inline database/setup_database_demo/create_role_and_db.sql]: missing newline at the end of the file

File: database/setup_database_demo/create_table_and_users.sql [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+ALTER TABLE "User" OWNER TO admin;
+
+COPY "User" (id, name, password, roles) FROM stdin;
+61d7f2e3-3387-4115-895e-b04a49967b27	admin	0f3477f71c65c7ac;7e21f7fe996f797973df579c9b2116b9de5cb7756cbbee30fca8a9a927d78854	["administrator"]
+75cb3dae-2a18-4112-8042-cf69198d1639	exampleuser	e297c78cdfd696a8;0d047fbfb5d090b37f1ba35faff745440df0ea6777d96a6733e4e420b910a66e	["user"]
+9de0c24b-e02e-4dc1-a5f7-3cdb1f474f7e	operator	3370ab41862a183c;17c1f3cd85ee7f5ae27e9f00e0ddd907151bf10ba0e90549c78f68fa7f93896f	["operator"]
+\.
+
+ALTER TABLE ONLY "User"
+    ADD CONSTRAINT "User_pkey" PRIMARY KEY (id);
```
Comment (heikomat) [inline database/setup_database_demo/create_table_and_users.sql]: missing newline at the end of the file

File: database/setup_database_demo/setup_database_demo.sh [hunk 3]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+########################################
+# execute this script as user "postgres"
+########################################
+
+psql -f create_role_and_db.sql
+psql -d processengine -f create_table_and_users.sql
+exit
```
Comment (heikomat) [inline database/setup_database_demo/setup_database_demo.sh]: missing newline at the end of the file

File: database/setup_database_demo/create_role_and_db.sql (hunk 1 above)
Comment (SimonSchneiderHSM) [reply database/setup_database_demo/create_role_and_db.sql]: done.

File: database/setup_database_demo/create_table_and_users.sql (hunk 2 above)
Comment (SimonSchneiderHSM) [reply database/setup_database_demo/create_table_and_users.sql]: done.

File: database/setup_database_demo/setup_database_demo.sh (hunk 3 above)
Comment (SimonSchneiderHSM) [reply database/setup_database_demo/setup_database_demo.sh]: done.

**Comment by SimonSchneiderHSM** [comment]
Thanks for the review. Next time I will use the pull request template.

Sorry, I don't have time to look into Windows installation as well. I work in a Ubuntu 17.04 VM now.

File: database/README.md.save [hunk 4]
```diff
@@ -0,0 +1,82 @@
+# process-engine Skeleton PostgreSQL-DB
```
Comment (heikomat) [inline database/README.md.save]: You accidentialy added a backup of the readme to your commit. Please remove it

File: database/setup_database_demo/create_table_and_users.sql [hunk 5]
```diff
@@ -16,4 +16,5 @@ COPY "User" (id, name, password, roles) FROM stdin;
 \.
 
 ALTER TABLE ONLY "User"
-    ADD CONSTRAINT "User_pkey" PRIMARY KEY (id);
\ No newline at end of file
+    ADD CONSTRAINT "User_pkey" PRIMARY KEY (id);
+
```
Comment (heikomat) [inline database/setup_database_demo/create_table_and_users.sql]: while the other files are fine now, you have added one newline too much at the end of this file. Every file should end with exactly one empty line

**Comment by SimonSchneiderHSM** [comment]
done.

**Comment by SimonSchneiderHSM** [comment]
done.

**Pull request merged**